domLab2.js

```diff
@@ -0,0 +1,460 @@
+//             },
+//             {
+//                 "name": "Geonosis",
+//                 "rotation_period": "30",
+//                 "orbital_period": "256",
+//                 "diameter": "11370",
+//                 "climate": "temperate, arid",
+//                 "gravity": "0.9 standard",
+//                 "terrain": "rock, desert, mountain, barren",
+//                 "surface_water": "5",
+//                 "population": "100000000000",
+//                 "residents": [
+//                     "https://swapi.co/api/people/63/"
+//                 ],
+//                 "films": [
+//                     "https://swapi.co/api/films/5/"
+//                 ],
+//                 "created": "2014-12-10T12:47:22.350000Z",
+//                 "edited": "2014-12-20T20:58:18.437000Z",
+//                 "url": "https://swapi.co/api/planets/11/"
+//             }
+//         ]
+//       }
+//       let results = obj["results"]
+//       let newArr = results.map(planet => {
+//           return [planet.name, planet.population]
+//       })
+//       let ul = document.createElement("ul")
+//       newArr.forEach(planet =>{
+//           let li = document.createElement("li")
+//           li.innerText = `${planet[0]} has ${planet[1]}`
+//           ul.appendChild(li)
+//       })
+//      document.body.appendChild(ul)   
+    
+// })
+
+//Question 5 
+
+// const createList = () => {
+//     let strElement = document.querySelector("#id")
+//     let string = strElement.value
+//     let ul = document.querySelector("#list1")
+//     let li = document.createElement("li")
+//     li.innerText = string
+//     ul.appendChild(li)
+
+// }
+
+// document.addEventListener("DOMContentLoaded", ()=>{
+    
+// })
+
+//Question 6 
+
+// const makeCopies = () => {
+//     let strElement = document.querySelector("#id")
+//     let strElement2 = document.querySelector("#idNum")
+//     let string = strElement.value
+//     let num = Number(strElement2.value)
+//     let ul = document.querySelector("#list")
+//     for(let i = 0; i < num; i++){
+//         let li = document.createElement("li")
+//         li.innerText = string
+//         ul.appendChild(li)
+//     }
+// }
+
+//Question 7
+
+const createPara = () => {
+    let strElement = document.querySelector("#id")
+    let ul = document.querySelector("#list")
+    let num = Number(strElement.value)
+    let li = document.createElement("li")
+    li.innerText = num
+    ul.appendChild(li)
+    let liArr = document.querySelectorAll("li")
+    let liValue = []
+    liArr.forEach(li => {
+        liValue.push(Number(li.innerText))
+    })
+    console.log(liArr)
+    let para1 = document.querySelector("#para1")
+    para1.innerText = average(liValue)
+}
+
+const average = arr => {
+    let sum = 0 
+    arr.forEach(num => {
+        sum += num
+    })
+    return sum/arr.length
+}
+
+const max = arr => {
+    let max = -Infinity
+    let min = Infinity
+    arr.forEach(num =>{
+
```

Comment:
There is `Math.max(num1,num2)` that takes numbers and returns the biggest number between them. It might be fun trying this out with this.